views/js/lib/mathquill/mathquill.js

```diff
@@ -4204,6 +4204,53 @@ define(['jquery'], function(jQuery) {
     LatexCmds.overline = LatexCmds.bar = bind(Style, '\\overline', 'span', 'class="mq-non-leaf mq-overline"');
     LatexCmds.overrightarrow = bind(Style, '\\overrightarrow', 'span', 'class="mq-non-leaf mq-overarrow mq-arrow-right"');
     LatexCmds.overleftarrow = bind(Style, '\\overleftarrow', 'span', 'class="mq-non-leaf mq-overarrow mq-arrow-left"');
+    LatexCmds.overleftrightarrow = bind(Style, '\\overleftrightarrow', 'span', 'class="mq-non-leaf mq-overleftrightarrow"');
+    LatexCmds.overset = bind(Style, '\\overset', 'span', 'class="mq-non-leaf mq-overset"');
+
+
+    LatexCmds.stackrel = P(MathCommand, function(_, super_) {
```

Comment:
It is a clever solution, and may even be the best one.

Although, this file is a third-party library, copied here as-is (or as it says at the top of the file, with minor modifications). By modifying it here, we would risk losing all your additions if we will ever update the lib.

I think there are a few options:
- Look for a way to extend `LatexCmds` from outside? I guess impossible, because it is private.
- Make `LatexCmds` non-private, by assigning `MathQuill.LatexCmds = LatexCmds` as our only modification of the file (document it too)?
- Add comment blocks around your changes, and extend our comment at the top of the file.
- We already created a [fork of mathquill under oat-sa](https://github.com/oat-sa/mathquill). We might put these modifications there, although the repo looks like a mess and I'm not sure which branches should be considered dead.